packages/medusa/src/api-v2/store/products/query-config.ts

```diff
@@ -0,0 +1,42 @@
+export const defaultStoreProductFields = [
+  "id",
+  "title",
+  "subtitle",
+  "description",
+  "handle",
+  "is_giftcard",
+  "discountable",
+  "thumbnail",
+  "collection_id",
+  "type_id",
+  "weight",
+  "length",
+  "height",
+  "width",
+  "hs_code",
+  "origin_country",
+  "mid_code",
+  "material",
+  "created_at",
+  "updated_at",
+  "*type",
+  "*collection",
+  "*options",
+  "*options.values",
+  "*tags",
+  "*images",
+  "*variants",
+  "*variants.options",
+  "*sales_channels",
```

Comment:
Is this useful for the storefront?

Reply:
Will double check, but I don't think its necessary. Good catch!